experimental/packages/opentelemetry-instrumentation-grpc/src/utils.ts

```diff
@@ -95,3 +95,18 @@ export const _methodIsIgnored = (
 
   return false;
 };
+
+/**
+ * Return method and service values getting from grpc name/path
+ * @param name the grpc name/path
+ */
+export const _extractMethodAndService = (name: string): { service: string, method: string } => {
```

Comment:
Might be worth adding some tests around this as string manipulations are often quite opaque.

Reply:
ok, sounds reasonable

Reply:
Done